src/test/java/no/sikt/sws/PrefixStripperTest.java

```diff
@@ -28,27 +25,24 @@ public class PrefixStripperTest {
     /**
     **  Merge all testcases into one stream.
     **/
-    static Stream<TestCaseSws> allRequestArguments() {
-        final ReadableInstant before = new Instant();
-        logger.info("Test cases loading");
-        var streamBuilder = Stream.<TestCaseSws>builder();
-
-        loadTestCases(streamBuilder, "proxy/requests-mapping.json");
-        loadTestCases(streamBuilder, "proxy/requests-search.json");
-        loadTestCases(streamBuilder, "proxy/requests-bulk.json");
-        loadTestCases(streamBuilder, "proxy/requests-doc.json");
-        loadTestCases(streamBuilder, "proxy/requests-indexes.json");
-        loadTestCases(streamBuilder, "proxy/requests-alias.json");
-
-        logger.info("loaded -> {} ms.", new Period(before,new Instant()).getMillis());
+    static Stream<TestCaseSws> getStreamOfTestCases() {
+        var streamBuilder = new TestCaseLoader.Builder();
+
+        streamBuilder.loadResource("proxy/requests-mapping.json");
+        streamBuilder.loadResource("proxy/requests-search.json");
+        streamBuilder.loadResource("proxy/requests-bulk.json");
+        streamBuilder.loadResource("proxy/requests-doc.json");
+        streamBuilder.loadResource("proxy/requests-indexes.json");
+        streamBuilder.loadResource("proxy/requests-alias.json");
```

Comment:
Here you could have used 

return new TestCaseLoader.Builder()
.loadResoruces()
.loadResources()
.build()